### ISSUE TITLE
2.8.17 : Problem Action URL in CustomView

### ISSUE THREAD

**Issue by mvisini** [centreon/centreon#6041]
**Centreon Web version**: 2.8.17

**Centreon Engine version**: 1.8.1

**Centreon Broker version**: 3.0.12

**OS**: Centos 6.9

Bonjour, 

encore un autre bug depuis la MAJ. Nous avons définis dans nos Templates d'hôtes des _action url_ de type : $protocol://$hostaddress (exemple : rdp://$hostaddress).

Cela fonctionne parfaitement dans le menu _Monitoring > Status Details_ : 
![image](https://user-images.githubusercontent.com/19703479/35798396-fa568870-0a62-11e8-9259-75bd2eb537dd.png)

En revanche dans les _CustomViews_ le lien généré est erroné :
![image](https://user-images.githubusercontent.com/19703479/35798491-3d625d56-0a63-11e8-9662-3957c234ecb9.png)

Cela n'était pas le cas en 2.7.8 avant la maj.

Mathieu

**Comment by lpinsivy** [comment]
Hi,

To help the whole community, could you write your issue in english?

Can you also provide the name of impacted Centreon widgets, the actual versions and the previous versions of them?

Regards,

**Comment by lpinsivy** [comment]
Ok this fix create your issue: https://github.com/centreon/centreon-widget-service-monitoring/commit/1639f7f761376633a583f12e31906a350f07730c

**Comment by mvisini** [comment]
Ok, but my file has already these line (included in a release ?).

**Comment by lpinsivy** [comment]
Yes you have to revert this commit (remove right part in green and add red left part).

**Comment by mvisini** [comment]
Ok it works ! Thanks
